src/aap_eda/services/ruleset/activate_rulesets.py

```diff
@@ -297,13 +293,7 @@ def _restart_activation(
             logger.info(msg)
 
         activation_db_logger.write(msg)
-        enqueue_restart_task(
-            seconds,
-            activation.id,
-            self.deployment_type,
-            self.ws_base_url,
-            self.ssl_verify,
-        )
+        enqueue_restart_task(seconds, activation.id)
```

Comment:
We still need `deployment_type` here, right? Its definition has 3 parameters

Reply:
Thanks for the warn, I forgot to push my last commit where I change the function to take the deployment type from the global config.

Reply:
Should we remove deployment_type from parameter list too?

Reply:
I mean we can alsoe remove deployment_type from here:
https://github.com/ansible/eda-server/pull/329/files#diff-3aa1d8dc47ee0af34b544249cc19758b5919dbddf7057e84dc032f49f6235f33L108

Reply:
Agree, I wanted to start this refactor at the scope of the enqueue_restart_task, to keep an small PR and doing this kind of refactors in small chunks. Do you agree to address it in later PR's?